.travis.yml

```diff
@@ -1,18 +1,28 @@
 language: python
-python:
-  - "2.7"
-  - "3.5"
+env:
+    # Enable python 2 and python 3 builds
+    - PYTHON=python2.7 PIP=pip2.7 PYSUF='2.7'
+    - PYTHON=python3 PIP=pip3 PYSUF=3
```

Comment:
Why this change? To add the `PYSUF` env variable?
If this is the reason, you could use `TRAVIS_PYTHON_VERSION` instead, which is defined automatically by Travis.